README.md

```diff
@@ -1,8 +1,59 @@
-# vscode-aks-tools
+# Azure Kubernetes Service (AKS) Extension for Visual Studio Code (Preview)
 
-* View your AKS clusters in the Kubernetes extension cloud explorer
-* Add AKS clusters to kubeconfig via the GUI
-* Run AppLens detector diagnostics on your AKS clusters
+The Visual Studio Code extension for Azure Kubernetes Service helps you view your AKS cluster, merge and save kubeconfig, perform AKS Diagnostics checks, and run AKS Periscope within VS Code.
+
+## Prerequisites
+
+Ensure that you have installed the [Kubernetes Tools](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools) extension for VS Code prior to installing the Azure Kubernetes Service Extension for Visual Studio Code following these [instructions](https://code.visualstudio.com/docs/azure/kubernetes#_install-the-kubernetes-extension).
```

Comment:
This should happen automatically if they install from the marketplace.  I wouldn't call it out except in a 'contributing' section.  (Similarly on the k8s extension we don't ask them to explicitly install the YAML extension.)

If you do want to explicitly list extension dependencies, I think this also depends on an Azure Account extension - Tats will be able to give you the reference.

Reply:
This is good to know. Let me remove the Prerequisites section then.